src/components/AccountSettings.vue

```diff
@@ -0,0 +1,135 @@
+<!--
+  - @copyright 2019 Christoph Wurst <christoph@winzerhof-wurst.at>
+  -
+  - @author 2019 Christoph Wurst <christoph@winzerhof-wurst.at>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<template>
+	<AppSettingsDialog
+		:open="true"
+		first-selected-section="alias-settings">
+		<AppSettingsSection
+			:title="t('mail', 'Account settings')">
+			<h2>{{ t('mail', 'Account settings') }}</h2>
+		</AppSettingsSection>
+		<p>
+			<strong>{{ displayName }}</strong> &lt;{{ email }}&gt;
+			<a
+				v-if="!account.provisioned"
+				class="button icon-renamesettings"
+				href="#account-form"
+				:title="t('mail', 'Change name')" />
+		</p>
```

Comment:
Can this block go into an appsettingsection of its own?

Reply:
yes, fixed